demos/applications/lis3dhtr.cpp

```diff
@@ -12,24 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <libhal-stm-imu/lis3dhtr.hpp>
 #include <libhal-util/serial.hpp>
 #include <libhal-util/steady_clock.hpp>
 
 #include "../hardware_map.hpp"
 
-hal::status application(hardware_map& p_map)
+hal::status application(hal::stm_imu::hardware_map& p_map)
 {
   using namespace std::chrono_literals;
   using namespace hal::literals;
 
   auto& clock = *p_map.clock;
   auto& console = *p_map.console;
+  auto& i2c = *p_map.i2c;
 
-  hal::print(console, "Demo Application Starting...\n\n");
+  hal::print(console, "Starting lis3dhtr Application...\n");
+  (void)hal::delay(clock, 50ms);
```

Comment:
```suggestion
  hal::delay(clock, 50ms);
```

ignoring the return type of delay is not necessary anymore.